tree/ntuple/v7/doc/specifications.md

```diff
@@ -251,7 +251,8 @@ Every field record frame of the list of fields has the following contents
 The field version and type version are used for schema evolution.
 
 If the flag 0x01 (_repetitive field_) is set, the field represents a fixed sized array.
-In this case, an additional 64bit integer specifies the size of the array.
+In this case, an additional 64bit integer specifies the size of the array. And
+one more (leaf) field description is expected to be found. 
```

Comment:
This is correct for `RArrayField` (a field type that catches the semantics of fixed-size arrays), but not required otherwise.  Thus, I would only mention that in Section `#### std::array<T, N> and array type of the form T[N]`.

Reply:
is the un-true part the word "leaf"? I thought in any case we'd have a meta-data field with `flags: 0x0001` and a repetition number, and then we will have another field description.

Reply:
You're right, the subfield doesn't necessarily need to be a leaf field (e.g., `std::array<std::vector<float>, 3>`). I think we currently have no case without a subfield (although the format doesn't forbid it).  I'd suggest

```
Typically, a sub field for the array content is expected to be found (see Section "Mapping of C++ Types to Fields and Columns").
```